File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cs
public class Solution {
    public int[] TopKFrequent(int[] nums, int k) {
        Dictionary<int, int> count = new Dictionary<int, int>();

        for (int i = 0; i < nums.Length; i++){
            if (count.ContainsKey(nums[i])){
                count[nums[i]]++;
            }
            else{
                count[nums[i]] = 1;
            }
        }
        int[] res = count.OrderByDescending(kvp => kvp.Value).Select(kvp => kvp.Key).Take(k).ToArray();
        
        return res;
        
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {

        int[] ans = new int[2*nums.length];
        int i = 0;
        while (i < nums.length*2){
            if (i > nums.length-1){
                ans[i] = nums[i - nums.length];
            }
            else{
                ans[i] = nums[i];
            }
            i++;

        }
        return ans;
        
    }
}